### DISCUSSION TITLE
Any good how-to for DOSBOX-X in DOS?  It loads but screen fills up with AUDIO errors

### DISCUSSION THREAD

**Discussion by keenerb** [joncampbell123/dosbox-x#4212] [General]
It does load under pure MSDOS on a PIII system I have.

The issue that makes it unusable is that screen fills with SDL windib audio warning: Timeout waiting for audio to complete, possible audio driver problem.  Forcing reset.

I have a soundblaster card working at I22, I5, D1 and I don't see any way to configure dosbox-x, and disabling all sound options in dosbox-x.ref isn't helping the issue.

**Comment by keenerb** [comment]
Apparantly dosbox doesn't load it's dosbox-x.ref config by default.  Once I passed -conf dosbox-x.conf as a command line option it started working decently.  Enabling sound causes it to crash but it's pretty functional otherwise.

**Comment by Torinde** [comment]
> Any good how-to for DOSBOX-X in DOS?

I need such as well - what I found so far:
- its README.TXT
- #4622
